modules/administration-guide/partials/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -7,62 +7,127 @@ To use Prometheus to collect, store, and query JVM metrics for {prod-short} Serv
+  name: che-host
+  namespace: {prod-namespace} <1>
+spec:
+  endpoints:
+    - interval: 10s <2>
+      port: metrics
+      scheme: http
+  namespaceSelector:
+    matchNames:
+      - openshift-devspaces
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: devspaces
 ----
-<1> The rate at which a target is scraped.
-<2> The rate at which the recording and alerting rules are re-checked.
-<3> The resources that Prometheus monitors. In the default configuration, a single job, `{prod-short} Server`, scrapes the time series data exposed by {prod-short} Server.
-<4> The scrape target for the metrics from port `8087`. Replace `__<{prod-short}_{orch-namespace}>__` with the {prod-short} {orch-namespace}. The default {prod-short} {orch-namespace} is `{prod-namespace}`.
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
+<2> The rate at which a target is scraped.
 ====
 
-. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
+. Create a Role and RoleBinding to allow Prometheus view the metrics
+
 +
-[source,terminal,subs="+attributes,quotes"]
+.Role
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
 ----
-$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prometheus-k8s
+  namespace: {prod-namespace} <1>
+rules:
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - ''
+    resources:
+      - services
+      - endpoints
+      - pods
 ----
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
+====
 
-.Verification
++
+.RoleBinding
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: view-openshift-monitoring-prometheus-k8s
+  namespace: {prod-namespace} <1>
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+----
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
+====
 
-. Use port forwarding to access the `Prometheus` Service locally:
+. Allow the in-cluster Prometheus instance to detect the ServiceMonitor in the {prod-short} namespace.
 +
+====
 [source,terminal,subs="+attributes,quotes"]
 ----
-$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
+oc label namespace {prod-namespace} openshift.io/cluster-monitoring=true
 ----
-. Verify that all targets are up by viewing the `targets` endpoint at `localhost:9090/targets`.
-. Use the Prometheus console to view and query metrics:
-** View metrics at `localhost:9090/metrics`.
-** Query metrics from `localhost:9090/graph`.
+====
+
+.Verification
+
+. In the *Administrator* view of the OpenShift web console, navigate to *Observe* -> *Metrics* from the sidebar menu on the left.
```

Comment:
Our style guide recommends avoiding positional references in the UI like "the sidebar menu on the left":
```suggestion
. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Metrics*.
```